src/snowflake/snowpark/session.py

```diff
@@ -790,13 +814,16 @@ def clear_packages(self) -> None:
         """
         self._packages.clear()
 
-    def add_requirements(self, file_path: str) -> None:
+    def add_requirements(self, file_path: str, force_push: bool = True) -> None:
         """
         Adds a `requirement file <https://pip.pypa.io/en/stable/user_guide/#requirements-files>`_
-        that contains a list of packages as dependencies of a user-defined function (UDF).
+        that contains a list of packages as dependencies of a user-defined function (UDF). Pure Python packages that
+        are not available on Anaconda will be pip installed locally and made available as an import (via zip file
+        on a remote stage).
```

Comment:
instead of Anaconda, I would rather say Snowflake because even though Anaconda might support it externally, Snowflake might decide to block certain packages.